module/userExtension/helper.js

```diff
@@ -12,13 +12,58 @@ const entitiesHelper = require(MODULES_BASE_PATH + '/entities/helper');
 const shikshalokamGenericHelper = require(ROOT_PATH + '/generics/helpers/shikshalokam');
 const elasticSearchData = require(ROOT_PATH + '/generics/helpers/elasticSearch');
 const programsHelper = require(MODULES_BASE_PATH + '/programs/helper');
-
+const userService = require(ROOT_PATH + '/generics/services/users');
+const kafkaClient = require(ROOT_PATH + '/generics/helpers/kafkaCommunications');
+const userExtensionsQueries = require(DB_QUERY_BASE_PATH + '/userExtensions');
 /**
  * UserExtensionHelper
  * @class
  */
 
 module.exports = class UserExtensionHelper {
+
+/**
+ * Create a Kafka event payload for a user-program-role mapping operation.
+ * @method
+ * @name createKafkaPayload
+ * @param {Object} userProfile - The user profile object containing `id` and `username`.
+ * @param {String} programId - The ID of the program being mapped.
+ * @param {String} role - The role being assigned or removed.
+ * @param {String} eventType - The type of event ('create' or 'delete').
+ * @returns {Object} - Kafka event payload.
+ */
+  static createKafkaPayload (userProfile, programId, role, eventType){
```

Comment:
@borkarsaish65 can we define createKafkaPayload, and addNewProgramEntry this outside of this module export, as function? because these codes no where using right? refer userProject helper in project service

Reply:
done